### ISSUE TITLE
Regarding Scale Factor 6.0 in Model Definition

### ISSUE THREAD

**Issue by Rodrigues-Royston** [fferflo/statewide-visual-geolocalization#5]
Thankyou for your exciting work! 
https://github.com/fferflo/statewide-visual-geolocalization/blob/main/geoloc/model/model.py

I saw in your model definition you have the scale factor as 6.0: 

 x = x / (jnp.linalg.norm(x, axis=-1, keepdims=True) + 1e-5)
x = x * scale

This has been done for forward pass in groundview as well as aerial view Models.
Will be grateful to learn why L2 normed features could not be used directly.

**Comment by fferflo** [comment]
The scale parameter implements the temperature parameter mentioned in 4.3 of the paper. Multiplying query and reference features by 6 is akin to dividing the logits by temperature=1/36 here: https://github.com/fferflo/statewide-visual-geolocalization/blob/main/geoloc/loss.py#L55

**Comment by Rodrigues-Royston** [comment]
Thanks to inform, I understand your response. 

I had an additonal quetion:
Are your models in double precision, float64 and not float32 ?

Will it make a difference if float32 is used for training? 

Thankyou very much for your exciting work and response.
